test/jats-reader.xml

```diff
@@ -1211,27 +1326,14 @@ These should not be escaped:  \$ \\ \&gt; \[ \{</preformat>
         </td>
         </tr>
       </tbody>
-    </table>
+    </table></table-wrap>
   </sec>
   <sec id="table-with-footer">
     <title>Table with footer</title>
-    <table>
+    <table-wrap><table>
       <col align="left" />
       <col align="left" />
       <col align="left" />
-      <tbody>
```

Comment:
For some strange reason, `<tfoot>` has to appear before `<tbody>`. Swapped them around.